pgstac_ingestor/services.py

```diff
@@ -26,18 +26,21 @@ def fetch_one(self, username: str, ingestion_id: str):
             raise NotInDb("Record not found") from e
 
     def fetch_many(
-        self, status: str, next: dict = None, limit: int = None
+        self,
+        status: str,
+        next: Optional[str] = None,
```

Comment:
`next` was set as a `dict` but I think it should be a `str`

Reply:
Reviewing [these docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.Pagination.html) I actually think it is a dict (although, it was previously typed incorrectly):

```suggestion
        next: Optional[Dict[str, Dict[str, str]]] = None,
```

(If we do ☝️, we'll need to add that `typing.Dict` import)